src/profile-logic/tracks.js

```diff
@@ -722,7 +776,8 @@ export function initializeSelectedThreadIndex(
 // order.
```

Comment:
is this comment still accurate?

Also can you please clarify whether this selects only global tracks or can also select local tracks?

Reply:
Updated it.